Documentation/observability/hubble/configuration/tls.rst

```diff
@@ -293,6 +293,255 @@ If you encounter issues after enabling TLS, you can use the following instructio
         ``*.{cluster-name}.hubble-grpc.cilium.io`` where ``{cluster-name}`` is
         the cluster name defined by ``cluster.name`` (defaults to ``default``).
 
+
+Validating the Installation
+---------------------------
+
+The following section guides you through validating that TLS is enabled for Hubble
+and the connection between Hubble Relay and Hubble Server is using mTLS to secure the session.
+Additionally, the commands below can be used to troubleshoot issues with your TLS configuration if you encounter any issues.
+
+Start by creating a Hubble CLI pod within the namespace that Hubble components are running in (for example: ``kube-system``):
+
+.. code-block:: shell-session
+
+    $ kubectl apply -n kube-system -f https://raw.githubusercontent.com/cilium/cilium/main/examples/hubble/hubble-cli.yaml
+
+List Hubble Servers by running ``hubble watch peers`` within the newly created pod:
+
+.. code-block:: shell-session
+
+    $ kubectl exec -it -n kube-system deployment/hubble-cli -- \
+    hubble watch peers --server unix:///var/run/cilium/hubble.sock
+
+    PEER_ADDED   172.18.0.2 kind-worker (TLS.ServerName: kind-worker.default.hubble-grpc.cilium.io)
+    PEER_ADDED   172.18.0.3 kind-control-plane (TLS.ServerName: kind-control-plane.kind.hubble-grpc.cilium.io)
+
+Copy the IP and the server name of the first peer into the following environment variables for the next steps:
```

Comment:
This step is assuming TLS is correctly setup, we should add a section showing the output of `hubble watch peers` when tls is disabled and what to look for (e.g. `TLS.ServerName` missing from the output). Maybe adding a step on the way to check whether hubble TLS is enabled in the cilium configMap since we are assuming access to `kube-system`?

Reply:
I've added a note just below this line mentioning the `TLS.ServerName` and to refer to previous steps to enable TLS if it's missing. I also added a step at the top of the troubleshooting section here to check the cilium-config as well.